wiki/Linux-Troubleshooting.md

```diff
@@ -29,6 +29,17 @@ If you have Amdvlk installed on your system, it overrides other vulkan drivers a
 
 Check if Amdvlk is installed by seeing if `ls /usr/share/vulkan/icd.d/ | grep amd_icd` shows anything. If so, uninstall Amdvlk from your system.
 
+## Failed to create VAAPI encoder (fedora)
+
+Blocky or crashing streams of gameplay and then an error window on your desktop saying:
+> Failed to create VAAPI encoder: Cannot open video encoder codec: Function not implemented. Please make sure you have installed VAAPI runtime.
+
+This seems to be an issue for AMD GPU fedora 39+ users, but maybe others.
+
+### Fix
+
+Switch from `mesa-va-drivers` to `mesa-va-drivers-freeworld`. [Guide on how to do so](https://fostips.com/hardware-acceleration-video-fedora/) or [the RPM docs](https://rpmfusion.org/Howto/Multimedia). Then reboot your machine.
```

Comment:
@The-personified-devil what do you think about keeping fostips here? i think keeping just to official sources better than to non-official guide

Reply:
Rpmfusion is also only official to itself, it's not an official project for fedora, I would add that it's the RPMFusion docs but I think it's fine to keep the other source since it's a lot more straightforward